tests/Aql/ExecutorTestHelper.h

```diff
@@ -284,6 +385,29 @@ struct ExecutorTestHelper {
   };
 
  private:
+  void testOutputBlock(SharedAqlItemBlockPtr const& outputBlock,
```

Comment:
I think this is not required anymore?
If you modified something in this part, please make sure to move it into the validateBlock asserter class above